[Communication Identity] Smoke tests failing

The samples for each package is used in our smoke tests, and some from the Communication Identity package have been failing since #13819 got merged

```
2021-02-18T20:41:57.0892703Z =========================================
2021-02-18T20:41:57.0893746Z SMOKE TEST FAILURES
2021-02-18T20:41:57.0894851Z Exception: TypeError: client.issueToken is not a function
2021-02-18T20:41:57.0902208Z ##[error]Test Failed - Package: communication-identity - Sample File:issueToken.js
2021-02-18T20:41:57.0904808Z {
2021-02-18T20:41:57.0905728Z   sample: {
2021-02-18T20:41:57.0907094Z     entrypoint: [AsyncFunction: main],
2021-02-18T20:41:57.0908548Z     name: 'communication-identity',
2021-02-18T20:41:57.0909823Z     sampleFile: 'issueToken.js',
2021-02-18T20:41:57.0911384Z     directory: '/home/vsts/work/1/s/sdk/communication/communication-identity/dist-samples/javascript'
2021-02-18T20:41:57.0912603Z   },
2021-02-18T20:41:57.0913518Z   result: {
2021-02-18T20:41:57.0914703Z     success: false,
2021-02-18T20:41:57.0915858Z     exception: TypeError: client.issueToken is not a function
2021-02-18T20:41:57.0917657Z         at Object.main [as entrypoint] (/home/vsts/work/1/s/sdk/communication/communication-identity/dist-samples/javascript/issueToken.js:34:34)
2021-02-18T20:41:57.0919382Z         at processTicksAndRejections (internal/process/task_queues.js:93:5)
2021-02-18T20:41:57.0920993Z         at async executeSample (/home/vsts/work/1/s/common/smoke-test/run.js:84:5)
2021-02-18T20:41:57.0922572Z         at async main (/home/vsts/work/1/s/common/smoke-test/run.js:42:18)
2021-02-18T20:41:57.0923707Z   }
2021-02-18T20:41:57.0924579Z }
2021-02-18T20:41:57.0925605Z Exception: TypeError: client.issueToken is not a function
2021-02-18T20:41:57.0926658Z {
2021-02-18T20:41:57.0927523Z   sample: {
2021-02-18T20:41:57.0928556Z     entrypoint: [AsyncFunction: main],
2021-02-18T20:41:57.0929821Z     name: 'communication-identity',
2021-02-18T20:41:57.0931261Z     sampleFile: 'revokeTokens.js',
2021-02-18T20:41:57.0932845Z     directory: '/home/vsts/work/1/s/sdk/communication/communication-identity/dist-samples/javascript'
2021-02-18T20:41:57.0934177Z   },
2021-02-18T20:41:57.0935095Z   result: {
2021-02-18T20:41:57.0936199Z     success: false,
2021-02-18T20:41:57.0937463Z     exception: TypeError: client.issueToken is not a function
2021-02-18T20:41:57.0940617Z         at Object.main [as entrypoint] (/home/vsts/work/1/s/sdk/communication/communication-identity/dist-samples/javascript/revokeTokens.js:35:42)
2021-02-18T20:41:57.0942411Z         at processTicksAndRejections (internal/process/task_queues.js:93:5)
2021-02-18T20:41:57.0944217Z         at async executeSample (/home/vsts/work/1/s/common/smoke-test/run.js:84:5)
2021-02-18T20:41:57.0947203Z         at async main (/home/vsts/work/1/s/common/smoke-test/run.js:42:18)
2021-02-18T20:41:57.0948302Z   }
2021-02-18T20:41:57.0949378Z }
2021-02-18T20:41:57.0950589Z Exc
```

--- a/sdk/communication/communication-identity/samples/javascript/issueToken.js
+++ b/sdk/communication/communication-identity/samples/javascript/issueToken.js
@@ -31,7 +31,7 @@ async function main() {
   console.log("Issuing Token");
 
   // Issue token and get token from response
-  const { token } = await client.issueToken(user, scopes);
+  const { token } = await client.getToken(user, scopes);
 
   console.log(`Issued token: ${token}`);
 }

--- a/sdk/communication/communication-identity/samples/javascript/revokeTokens.js
+++ b/sdk/communication/communication-identity/samples/javascript/revokeTokens.js
@@ -32,9 +32,9 @@ async function main() {
   console.log("Issuing Tokens");
 
   // Issue tokens
-  const { token: token1 } = await client.issueToken(user, scopes);
-  const { token: token2 } = await client.issueToken(user, scopes);
-  const { token: token3 } = await client.issueToken(user, scopes);
+  const { token: token1 } = await client.getToken(user, scopes);
+  const { token: token2 } = await client.getToken(user, scopes);
+  const { token: token3 } = await client.getToken(user, scopes);
 
   console.log("Issued tokens:");
   console.log(token1);

